.travis.yml

```diff
@@ -3,15 +3,11 @@ sudo: false
 
 matrix:
   include:
-    - php: 5.3
-      dist: precise
-    - php: 5.4
-    - php: 5.5
     - php: 5.6
     - php: 7.0
     - php: 7.1
     - php: 7.2
-    - php: hhvm
+    - php: 7.3
```

Comment:
Changing the project to use CircleCI instead of Travis shouldn't take more than 30 mins, so we should probably do this as well.

Reply:
CircleCi is working 👍